Apps/Sandcastle/gallery/Terrain.html

```diff
@@ -182,17 +183,25 @@
     }
 
     Cesium.when(Cesium.sampleTerrain(viewer.terrainProvider, 9, terrainSamplePositions), sampleTerrainSuccess);
+
+    var target = new Cesium.Cartesian3(300770.50872389384, 5634912.131394585, 2978152.2865545116);
```

Comment:
Instead of copying this code from the `onselect` function above, move that function and give it a name, and have both places call that function.